internal/route/api/v1/repo/commits.go

```diff
@@ -18,6 +19,52 @@ import (
 	"gogs.io/gogs/internal/gitutil"
 )
 
+// GetAllCommits returns a slice of commits starting from HEAD.
+func GetAllCommits(c *context.APIContext) {
+	if strings.Contains(c.Req.Header.Get("Accept"), api.MediaApplicationSHA) {
+		c.SetParams("*", c.Params(":sha"))
+		GetReferenceSHA(c)
+		return
+	}
```

Comment:
What is this part of the logic about?

Reply:
I think I might have forgot to remove this after copying from GetSingleCommit.